README.md

```diff
@@ -1,11 +1,74 @@
+[![Go Reference](https://pkg.go.dev/badge/github.com/ngrok/ngrok-go.svg)](https://pkg.go.dev/github.com/ngrok/ngrok-go)
+[![Go](https://github.com/ngrok/ngrok-go/actions/workflows/buildandtest.yml/badge.svg)](https://github.com/ngrok/ngrok-go/actions/workflows/buildandtest.yml)
 # ngrok-go
 
-**Note: This is alpha-quality software. Interfaces are subject to change
-without warning**
+ngrok is a globally distributed reverse proxy commonly used for quickly getting a public URL to a service running inside a private network, such as your laptop. The ngrok agent is usually deployed inside a private network and is used to communicate with the ngrok cloud service.
```

Comment:
I think "such as your laptop's network." is more grammatical.

> a service running inside a private network, such as your laptop.

It's hard to tell if "such as" is a modifier for just the noun phrase "private network", or for "a service ....", but in either case, I think we want a different phrasing than just laptop.

I think our options are "such as your laptop's network" (making it obvious it's the network), or "on your laptop" (binding to "a service").